lms/templates/learner_dashboard/course_card.underscore

```diff
@@ -1,13 +1,20 @@
-<img class="header-img" src="<%- image_url %>" alt="<%- display_name %>"/>
-<div class="course-details">
-    <h3 class="course-title"><%- display_name %></h3>
-    <p class="course-key"><%- key %></p>
-    <% if(enrollment_status){ %>
-        <span class="enrollment-info"><%- course_start %> - <%-course_end %></span>
-        <a href="<%- courseware_url %>" class="course-link cta"><%- gettext('View Course') %></a>
-    <% }else{ %>
-        <span class="enrollment-info"><%- gettext('Not Yet Enrolled') %></span>
-        <a href="<%- marketing_url %>" class="course-link cta"><%- gettext('Learn More') %></a>
-    <% } %>
+<div class="grid-container grid-manual">
+    <a href="<%- marketing_url %>" class="course-image-link">
+        <img class="header-img col-12 xl-col-3 lg-col-3 md-col-3 sm-col-12" src="<%- image_url %>" alt="<%- display_name %>"/>
```

Comment:
I think that would make sense, yes. My only hesitation is that the same link is present for the heading. @cptvitamin what should the behavior be here? We have a course image and course title heading, both of which link to the marketing site's course details page. Should the image and link be hidden from screenreaders and some additional sr text added to the h3 element to provide context (i.e. {course-title} Home Page) or should that be added to the image alt so we have 2 links going to the same place?